chiselFrontend/src/main/scala/chisel3/Clock.scala

```diff
@@ -30,6 +31,10 @@ sealed class Clock(private[chisel3] val width: Width = Width(1)) extends Element
   /** Not really supported */
   def toPrintable: Printable = PString("CLOCK")
 
+  /** Returns the contents of this wire as a [[scala.collection.Seq]] of [[Bool]]. */
```

Comment:
I'm confused. `asBool` returns a `Seq`? Shouldn't the method be `asBools`?

Reply:
I think the method is as intended, but the comment is wrong

Reply:
I've updated the comment. If others approve, so do I.